src/util/cleanDu/cleanImmediateDus.js

```diff
@@ -0,0 +1,62 @@
+const fn = require('../../config/fieldNames')
+const getIrrelevantDus = require('./getIrrelevantDus');
+const deleteDus = require('./deleteDus');
+const { logLevel } = require('../logger');
+const logDetails = require('../logDetails');
+const p =  require('../../config/paths');
+const getRawData = require('../getRawData');
+const moment = require('moment');
+//const Set = require("./collections/set");
+const preRun = require('../preRun');
+//receives dataSource,identifiers an object with 3 possible fields, ...
+//fetches all possible people with the received identifiers from kartoffel, then for each person
+//iterates over all of their domainUsers and checks if it is consistent with the databases (pncy)
+//if the person is still in the database of that datasource then we continue with no action.
+//otherwise we remove the domainUser from the person in kartoffel
+async function cleanImmediateDus(dataSource, identifiers, sendLog, Auth) {
+
+    //first: get the dataSourcePersons from kartoffel
+    let dataSourcePersons = new Set();
+    if (identifiers.identityCard){
```

Comment:
for in - identifier for both pn and id

Reply:
cant do "for await in" like the "for await of"